app/pug/user_profile.pug

```diff
@@ -52,12 +50,11 @@ block content
       
       dt Roles
       dd
-        pre #{JSON.stringify(user.roles)}
-    
+        codelike #{JSON.stringify(user.roles)}
+      
       dt Permissions granted by these Roles:
       dd
-        <pre>#{JSON.stringify(user.permissions)}</pre>
-    
+        codelike #{JSON.stringify(user.permissions).replace(/,/g, ", ")}
```

Comment:
As mentioned below we can’t invent new HTML elements.  Adding a dot in front would establish this as a class

Reply:
Changed in new commit.